lib/utils/annotation.go

```diff
@@ -0,0 +1,36 @@
+package utils
+
+import (
+    "fmt"
+    metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+    "k8s.io/client-go/kubernetes"
+    "k8s.io/client-go/tools/clientcmd"
+    "utils"
+)
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+const TemplateAnnotation = "projectsveltos.io/template"
+
+func IsTemplate(obj metav1.Object) bool {
+    annotations := obj.GetAnnotations()
+    if annotations != nil {
+        _, ok := annotations[TemplateAnnotation]
+        return ok
+    }
+    return false
+}
+
+func main() {
```

Comment:
We don't need a main. 

This library is used by other sveltos components. Those will import libsveltos and use IsTemplate method you added.